### PR TITLE
css-brochure-form | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
image

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: "Brandon Grotesque", sans-serif;
+}
+
+body {
+  margin-top: 40px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You should not really apply margin to the body, you don't want to push the entire content of the page away from the viewport. The `body` _is_ the content of the viewport :joy:. You should use padding instead.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  color: white;
+  font-size: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  width: 90px;
+  height: 35px;
+  border-width: 0;
+  border-radius: 20px;
+  margin-top: 10px;
+  margin-left: 380px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: WOAH! That's a lot of margin! I'm assuming you are using this to push the element to the right side of the form, yes? Margin should only be used for minor adjustments on elements, not as a layout tool. For something like this, you could simply set the `div` element that is the parent of the `button` to have `text-align` set to `right`, and the button will shift to the right side of the container.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Angel! Great job so far, please see my notes

File: css-brochure-form/styles.css (hunk 1 above)
Comment (angelsleung) [reply css-brochure-form/styles.css]: Yes, that makes sense! Thanks Cody!!

File: css-brochure-form/styles.css (hunk 2 above)
Comment (angelsleung) [reply css-brochure-form/styles.css]: Ahh so that's how you do it, gotcha!! 😃

**Comment by angelsleung** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**